src/converters/lintConfigs/rules/ruleConverters/eslint-plugin-sonarjs/no-return-type-any.ts

```diff
@@ -0,0 +1,11 @@
+import { RuleConverter } from "../../ruleConverter";
+
+export const convertNoReturnTypeAny: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "@typescript-eslint/no-unsafe-return",
```

Comment:
`@typescript-eslint/no-unsafe-return` is actually a good bit more complex than `no-return-type-any`; it uses the type checker to check _inferred_ `any` return types as well.

I think this rule might not have an equivalent anymore?

Reply:
Yes, you are right. But I think that the two rules have the same intention.
I dont know what should we do with this. Remove this converter or maybe just set to `"warn"`?

Reply:
Generally we'd remove the converter, yeah. `tslint-to-eslint-config` will know to print a notice saying _"we don't know of an equivalent"_ and keep this rule in the list of rules to still be run with the [`eslint-plugin-tslint` adapter](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin-tslint).

Reply:
I removed the converter.